replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+suppressPackageStartupMessages(require(aod)) # wald.test
+suppressPackageStartupMessages(require(broom))
+suppressPackageStartupMessages(require(lsr)) #etaSquared
+suppressPackageStartupMessages(require(ggplot2))
+suppressPackageStartupMessages(require(foreign))
+
+# detect the user's directory:
+directory  <- "C:/Users/Nick/Documents/Github/scarcityvalueRep"
+if (Sys.getenv('USER')=="air") directory <- "~/Dropbox/SOC504/replicate"
+setwd(directory)
+
+# Function : calculates standard error of the mean
+Sem <- function(x) {
+  sqrt(var(x)/length(x))
+  }
+
+# Function : to generate mean and confidence interval & name
+Mean_CI <- function(title, x){
+  print(title, quote=F)
+  # print Mean
+  M <- t.test(x) %>%
+    .$estimate %>%
+    round(digits = 2) %>%
+    .[[1]]
+  print(c("M:", M), quote = F)
+  # print Confidence Interval 95%
+  CI <- t.test(x) %>%
+    .$conf.int %>%
+    round(digits = 3) %>%
+    .[1:2]
+  print(c("CI : ", CI), quote = F)
+}
+```
+<br> <br> <br>
+*note: beta coefficients for linear regressions calculate consistantly larger than published, even though the t value and p values are as published.  This occurs several times throughout. t values that match the publication are available in the models.*
+<br> <br>
+*Update 4/25/15 - The unmatched beta coefficients are due to the publication reporting standardized coefficients, and R function lm reporting unstandardized beta coefficients*
+<br> <br>
+*Update 4/26/15 - unmatched beta coefficients have been fixed using the scale() function.*
+
+### Study 1a
+
+
+> In this study, there were 103 participants (mean age = 29.3 years; median household size = 3 people; median household income = $35,000). One participant was excluded as an outlier on income (more than 3 SD from the mean). We conducted binary logistic regressions to compare how frequently participants cited each of the two considerations of interest (trade-offs vs. location) as a function of income.
+
+```{r echo = FALSE}
+# Data import
+
+# read in the data for 1a using the spss file
+# the warning/messages suppressed; the data imports cleanly.
+suppressMessages(suppressWarnings(raw_1a_spss <- read.spss("framing/study_1a.sav", to.data.frame = TRUE)))
+suppressMessages(suppressWarnings(raw_1a_csv <- read.csv("study_1a_demog.csv", header = FALSE)))
+
+#calculate mean age
+raw_1a_csv %>%
+  summarise(average_age = mean(V3)) %>%
+  round(digits = 1)
+
+#calculate sex breakdown
+raw_1a_csv %>%
+  group_by(V4) %>%
+  summarise(sex_breakdown = n())
+
+#calculate sample size, median household income, median household size
+raw_1a_spss %>%
+  summarise(sample.size = n(), median.household.income = median(income2), median.household.size = median(house))
+
+clean_1a <- raw_1a_spss %>%
+  filter(filter_. != "Not Selected")
+```
+
+> Higher-income participants were more likely than lower-income participants to name location as the main consideration, β = 2.52, Wald-test χ2(1, N = 102) = 5.54, p < .05,
```

Comment:
I think you can take advantage of R Markdown here and make these values of the betas, chi-square statistic, etc., dynamic (so if anything in the data set changes, these values change along with it; and you can copy/paste statements like these in other project reports without a lot of editing)